src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -198,7 +199,8 @@ public void updateStudentCascade(String originalEmail, StudentAttributes student
         StudentAttributes originalStudent = getStudentForEmail(student.course, originalEmail);
         updateStudentCascadeWithSubmissionAdjustmentScheduled(originalEmail, student, hasDocument);
         
-        /* finalEmail is the string to be used to represent a student's email.
+        /*
+         * finalEmail is the string to be used to represent a student's email.
          * This is because:
          *  - originalEmail cannot be used when student's email is being updated with a new valid email
```

Comment:
The markdown style formatting will not work as you expect. You can try to use html tag to format the javadoc.
More [here](http://www.oracle.com/technetwork/articles/java/index-137868.html).